### DISCUSSION TITLE
Exclude authors and PR number from automatically generated release notes

### DISCUSSION THREAD

**Discussion by RobertBlaauboer** [community/community#10818] [Other Feature Feedback, Questions, & Ideas]
### Describe the feature or problem you’d like to solve

Being able to automatically generate release notes is great. 
Even more so if they could be in a format that you could send to non-git users.

Currently a PR could be formatted as follows:

Exciting New Features 🎉
- Add support for third-party tool by @ContributorName in `#20`

Here, the PR name is followed by the contributor's name and the pull request number.
However, non-git users don't need information on who developed a feature (or the PR number) and 
would only need:

Exciting New Features 🎉
- Add support for third-party tool

This would allow us to directly send clear release notes to (beta)users.

### Proposed solution
An option in .github/release.yml that specifies how the PR should be shown in the generated release notes.

**Comment by sanders9** [comment]
Hello. Any news about this point?

**Comment by jamesfoster-excelpoint** [comment]
This would be a nice addition to the config options in `.github/release.yml` :+1:
